src/explore-education-statistics-frontend/server.js

```diff
@@ -1,4 +1,10 @@
 require('core-js/fn/array/flat-map');
+require('core-js/fn/array/flatten');
+
+if (!Array.prototype.flat) {
```

Comment:
Does babel not handle this kind of rewriting for browsers / versions of Node that don't provide Array.prototype.flat?

Reply:
Nope, we don't invoke Babel for our Node code. Babel doesn't necessarily polyfill `flat`/`flatMap` without some additional configuration anyway.